src/ethereum/EthereumVerifier.js

```diff
@@ -230,7 +231,7 @@ class EthereumVerifier {
       const taskId = task.getTaskId();
 
       try {
-        res.taskParams = await this._contractApi.getTaskParams(taskId);
+        res.taskParams = await this._contractApi.getTaskParams(nodeUtils.add0x(taskId));
```

Comment:
Why are you adding the `0x` here, as well as in the `EnigmaContractReaderAPI` and `EnigmaContractWriterAPI`?

Reply:
You are correct, that's an oversight on my part. Corrected in d307806